### PR TITLE
Add KlipperScreen to kiauh

### PR DESCRIPTION
**Pull request by jordanruthe**
Add functions to support KlipperScreen installation/removal/updating from kiauh

### CODE REVIEW

File: scripts/update.sh [hunk 1]
```diff
@@ -166,3 +169,20 @@ update_moonraker(){
...
+  git checkout -f origin/master && ok_msg "Checkout successfull"
+  #KLIPPERSCREEN_NEWREQ_MD5SUM=$(md5sum $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt)
+  if [[ $(md5sum $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt | cut -d " " -f1) != $KLIPPERSCREEN_OLDREQ_MD5SUM ]]; then
+    status_msg "New dependecies detected..."
+    PYTHONDIR="${HOME}/.KlipperScreen-env"
+    $PYTHONDIR/bin/pip install -r $KLIPPERSCREEN_DIR/scripts/KlipperScreen-requirements.txt
+    ok_msg "Dependencies have been installed!"
+  fi
+  ok_msg "Update complete!"
+  start_klipper
```
Comment (th33xitus) [inline scripts/update.sh:187]: Do we maybe need a `start_klipperscreen `here instead?
I haven't seen a `stop_klipper `call before... but only a `stop_klipperscreen`.

File: scripts/remove.sh [hunk 2]
```diff
@@ -302,3 +302,29 @@ remove_nginx(){
...
+      status_msg "Removing KlipperScreen directory ..."
+      rm -rf $KLIPPERSCREEN_DIR && ok_msg "Directory removed!"
+    fi
+    if [ -d $KLIPPERSCREEN_ENV_DIR ]; then
+      status_msg "Removing KlipperScreen VENV directory ..."
+      rm -rf $KLIPPERSCREEN_ENV_DIR && ok_msg "Directory removed!"
+    fi
+    #remove KlipperScreen systemd file
+    if [ -e /etc/nginx/sites-available/mainsail ]; then
+      status_msg "Removing KlipperScreen configuration for Nginx ..."
```
Comment (th33xitus) [inline scripts/remove.sh:325]: The if condition here looks like a typo.
You probably wanted to check for `[ -e /etc/systemd/system/KlipperScreen.service ]`.

And the status_msg does not suit the action followed by it.
You probably want to call it `status_msg "Removing KlipperScreen Service ..."`

**Review by th33xitus — COMMENTED** [review]
Looks good to me so far, if you fix those two little things i found, i think we can merge it without an issue.

**Comment by jordanruthe** [comment]
Good catches, just pushed an update

File: scripts/remove.sh [hunk 3]
```diff
@@ -321,7 +321,7 @@ remove_klipperscreen(){
       rm -rf $KLIPPERSCREEN_ENV_DIR && ok_msg "Directory removed!"
     fi
     #remove KlipperScreen systemd file
-    if [ -e /etc/nginx/sites-available/mainsail ]; then
+    if [ -e /etc/systemd/system/KlipperScreen.service ]; then
       status_msg "Removing KlipperScreen configuration for Nginx ..."
```
Comment (th33xitus) [inline scripts/remove.sh:325]: status_msg still needs to be fixed. :)

File: scripts/update.sh [hunk 4]
```diff
@@ -184,5 +184,5 @@ update_klipperscreen(){
     ok_msg "Dependencies have been installed!"
   fi
   ok_msg "Update complete!"
-  start_klipper
+  start_klippererscreen
```
Comment (th33xitus) [inline scripts/update.sh:187]: old typo gone, new typo here haha

Comment (jordanruthe) [reply scripts/update.sh]: Backed that out and re-pushed it

**Pull request merged**